gatsby-config.js

```diff
@@ -10,21 +10,30 @@ module.exports = {
         short_name: `SeaSensors`,
         icons: [
           {
-            "src": "/android-chrome-192x192.png",
-            "sizes": "192x192",
-            "type": "image/png"
+            src: '/android-chrome-192x192.png',
+            sizes: '192x192',
+            type: 'image/png'
           },
           {
-            "src": "/android-chrome-512x512.png",
-             "sizes": "512x512",
-             "type": "image/png"
+            src: '/android-chrome-512x512.png',
+            sizes: '512x512',
+            type: 'image/png'
           }
         ],
         start_url: `/`,
         background_color: `#ffffff`,
         theme_color: `#ffffff`,
-        display: `standalone`,
-      },
+        display: `standalone`
+      }
     },
+    {
+      resolve: `gatsby-plugin-nprogress`,
+      options: {
+        // Setting a color is optional.
+        color: `tomato`,
```

Comment:
`tomato`? Is this the name of SeaSensors main/primary color? Otherwise switch to that.